client/lib/create-config/index.js

```diff
@@ -66,9 +66,38 @@ const config = ( data ) => ( key ) => {
 const isEnabled = ( data ) => ( feature ) =>
 	( data.features && !! data.features[ feature ] ) || false;
 
+/**
+ * Enables a specific feature.
+ *
+ * @param {string} feature Feature name
+ * @param {object} data the json environment configuration to use for getting config values
+ * @api public
+ */
+const enable = ( data ) => ( feature ) => {
+	if ( data.features ) {
```

Comment:
In dev mode should we consider throwing a warning in the `console` when the feature being enabled/disabled is not currently defined such as

```
if ( 'development' === process.env.NODE_ENV ) {
		throw new ReferenceError(
			`Could not find config value for key '${ key }'\n` +
				"Please make sure that if you need it then it has a default value assigned in 'config/_shared.json'"
		);
	}
```

This is what the primary `config` does already.